app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -55,7 +63,7 @@ function FieldButton({
       disabled={disabled}
       loading={isLoading}
       onClick={callback}
-      type="submit"
+      type={"button"}
```

Comment:
`isSubmit` should be used here (you probably have an unused variable lint error).